Game/Pregnancy/MenstrualCycle.gd

```diff
@@ -113,8 +125,14 @@ func processTime(seconds):
 		if(shouldOvulate()):
 			ovulate()
 	
+	var readyFetusAmount: = 0
 	for egg in impregnatedEggCells:
 		egg.processTime(seconds)
+		if(egg.fetusIsReadyForBirth()):
+			readyFetusAmount += 1
+	
+	if(readyFetusAmount == impregnatedEggCells.size() && impregnatedEggCells.size() > 0):
+		onAllFetusReadyForBirth()
```

Comment:
this makes it so this function is called on every process time call. Before it was called when one of the eggcells was ready. This could be kinda slow.

Reply:
I checked readyToGiveBirth signal is never used after this commit 
https://github.com/Alexofp/BDCC/commit/49602bb2e9056017593462cb35af3daabadce38b
So it is only a function call which do nothing after the first time it is called.